tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md

```diff
@@ -1,39 +1,227 @@
+
+[GCP Releases](https://console.cloud.google.com/storage/browser/vic-product-ova-releases?project=eminent-nation-87317&authuser=0)
+
+[VIC Engine Releases](https://console.cloud.google.com/storage/browser/vic-engine-releases?project=eminent-nation-87317&authuser=0)
+
+[OVA Smoke Test](https://confluence.eng.vmware.com/pages/viewpage.action?pageId=242300713)
+
+# Common Environment Setup:
+* These tests requires that a vCenter server is running and available with DRS enabled.
+
+# Test Template
+### Test Steps
+1. Deploy **[Deploy Old OVA Link]** previous version **[OVA Old Version]** of OVA of the vSphere Integrated Containers appliance
+2. Go to URL http://<appliance_IP> in browser
+3. Login to management portal using the credentials set during OVA deployment
+4. Go to **[Root Cert Path]** and download root certificate file
+5. Get respective version of VIC engine binaries
+6. Copy downloaded root cert to the same dir as vic engine binaries
+7. On vic engine binaries dir, run **[VCH Create Command]** to create VCH
+8. Go back to management portal, add newly created VCH as new host
+9. Add harbor registry to management portal
+    - Go to **[Add Registry Path]**
+    - Provide appliance IP as "Address"
+    -  Create new credentials for harbor and use it
+10. Copy root cert to docker dir
+    - `sudo mkdir -p /etc/docker/certs.d/<appliance_ip>`
+    - `sudo mv ca.crt /etc/docker/certs.d/<appliance_ip>/ca.crt`
+11. Tag and Push docker image to harbor registry
+    - `docker pull busybox`
+    - `docker tag busybox <appliance_ip>/<default_harbor_project>/busybox:test`
+    - `docker login <appliance_ip> --username <username_set_during_deployment> --password <password_set_during_deployment>`
+    - `docker push <appliance_ip>/<default_harbor_project>/busybox:test`
+12. Create containers from the image in harbor
+    - Go to **[Create Container Path]**
+    - Search for recently pushed image in harbor "<appliance_ip>/<default_harbor_project>/busybox:test" and create container using it
+    - Make sure container is running
+13. Create container from Docker hub
+    - Go to **[Create Container Path]**
+    - Search for photon image and create container using it
+    - Make sure container is running
+14. Shut down (don't power off) deployed version **[OVA Old Version]** of OVA
+15. Deploy **[Deploy New OVA Link]** new version **[OVA New Version]** of OVA of the vSphere Integrated Containers appliance (Don't power on yet)
+16. Follow procedure steps in **[Upgrade OVA Link]** to swap disks and upgrade OVA
+17. Power on new version of deployed OVA appliance and note down the IP
+18. Go to URL http://<appliance_IP> in browser
+19. Verify that harbor registry added in step 9 is available
+20. Verify that the image pushed in step 11 is available in harbor registry
+21. Verify that the containers created in steps 12 and 13 are still available and running
```

Comment:
We should also verify that the VCH added as a host in the older OVA should be present in the upgraded OVA.

Reply:
added